### PR TITLE
Flesh out PHPUnit

### PR DESCRIPTION
**Pull request by ggorlen**

### CODE REVIEW

**Review by kazk — APPROVED** [review]

File: content/languages/php/phpunit.md [hunk 1]
```diff
@@ -9,6 +9,28 @@ tags:
...
+    return $a + $b;
+}
+```
+
+```php
+class AddTest extends TestCase {
+    public function testAdd() {
+        $expected = 3;
+        $actual = add(1, 2);
+        $this->assertEquals($expected, $actual);
```
Comment (error256) [inline content/languages/php/phpunit.md:29]: Tests like this with non-strict equality can be passed by `return true;` for all non-zero values.

Comment (ggorlen) [reply content/languages/php/phpunit.md:29]: Sorry, I don't think I understand. Isn't `assertEquals` strict equality?

Comment (error256) [reply content/languages/php/phpunit.md:29]: No, strict equality is `assertSame`.

Comment (ggorlen) [reply content/languages/php/phpunit.md:29]: I'm still not following the consequence of this. Can you provide an example where `assertEquals` is problematic to use for CW?

[`assertSame`](https://phpunit.readthedocs.io/en/9.5/assertions.html#assertsame) seems to be identity of objects, not what I would want:

> Reports an error identified by `$message` if the two variables `$expected` and `$actual` do not reference the same object.

Comment (error256) [reply content/languages/php/phpunit.md:29]: It's problemattc pretty much with every function returning numbers:
https://www.codewars.com/kata/reviews/57b89247f07f80714400001e/groups/62adf0ca916c01000120c085

Comment (ggorlen) [reply content/languages/php/phpunit.md:29]: OK, thanks--that's pretty clearly broken. Great catch. So it seems that `assertEquals` is best for object equalty and `assertSame` for everything else, including arrays?

Comment (error256) [reply content/languages/php/phpunit.md:29]: It seems so, but I don't really know PHP, so I can't tell much.

Comment (ggorlen) [reply content/languages/php/phpunit.md:29]: Neither do I, clearly. I'll add a PR for that, reference this conversation and hopefully someone can validate it if there's any issues. Thanks again--super helpful.

**Pull request merged**